spikeinterface/curation/auto_merge.py

```diff
@@ -392,10 +393,11 @@ def compute_templates_diff(sorting, templates, num_channels=5, num_shift=5, pair
     return templates_diff
 
 
-class MockWaveformExtractor:
-    def __init__(self, recording, sorting):
-        self.recording = recording
-        self.sorting = sorting
+class MockWaveformExtractor(WaveformExtractor):
+    def __init__(self, we, sorting):
+        WaveformExtractor.__init__(self, recording=None, sorting=sorting, folder=None,
+                                   rec_attributes=we._rec_attributes, allow_unfiltered=True,
+                                   sparsity=None)
```

Comment:
I think this is better to not be a real WaveformExtractor but only to mock!
We should not put inheritance here.
I did this to be faster.
lets keep the MockWaveformExtractor independant until we have an intermediate.